### ISSUE TITLE
IOError: [Errno 2] No such file or directory: u'.locks/setup.lock|SN35393|27692|6373924406458409782'**

### ISSUE THREAD

**Issue by jiaozizhao** [cvlab-epfl/LIFT#3]
Hi, When I run the code, I got an issue like below. I am not sure about the issue because I can not find any similar one on Internet. Could you help me? Thank you very much.

![image](https://cloud.githubusercontent.com/assets/23083784/20149319/a830a2fc-a6a8-11e6-93cd-b68b15488ec1.png)

**Comment by kmyi** [comment]
Hi, can you try the new release? there was a hidden folder that was not added to the git repository.

Cheers,
Kwang

**Comment by jiaozizhao** [comment]
Really thank you very much for the new release. Now it can work well. 
But it seems that there is a limitation of the test image size. I tested my own image of size 256x256, an error occurred. And then I tested another image of size 1000x1000, it is really OK. Are there any parameters I need to adjust so that I can test an image of small size? 

And will you release training code for us users to finetune on our own data set?
Thank you very much.
